src/cmd/cli/command/compose.go

```diff
@@ -547,5 +547,7 @@ services:
 	composeCmd.AddCommand(makeComposeStartCmd())
 	composeCmd.AddCommand(makeComposeRestartCmd())
 	composeCmd.AddCommand(makeComposeStopCmd())
+	composeCmd.AddCommand(makeComposeEstimateCmd())
```

Comment:
I think we should not add `estimate` under `compose`. It's the first of such sub-commands that's not in `docker compose`.

Reply:
We should probably rename the make-func too:
```suggestion
```

Reply:
Sounds good to me 👍